src/mcp_server.py

```
@@ -0,0 +1,67 @@
+    :return:
+    """
+    try:
+        global api_url
+        api_url = os.getenv("API_URL")
+        if not api_url:
+            raise ValueError("API_URL environment variable is required.")
+        # Run the FastMCP server with STDIO transport
+        mcp.run()
+    except KeyboardInterrupt:
+        log("Server shutdown requested by user")
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)